.changeset/wild-ducks-bake.md

```diff
@@ -0,0 +1,5 @@
+---
+"chainlink": patch
+---
+
+#added FinalityDepth config where it was missing
```

Comment:
I think we dont need a change set for this, because this does not actually need to bump a new version & does not add a new feature & not modifying any existing functionality

Reply:
There is a check that fails if changeset is not added. Is it ok to have that check as failing before merging the PR?

Reply:
Hmm, I think so, can you please check with framework pod?

Reply:
Done.